### DISCUSSION TITLE
Min Selectable Temperature

### DISCUSSION THREAD

**Discussion by Rev-777** [revk/xESP32-Faikout#535] [Q&A] [answered]
Hi RevK,

Thanks for the project, I have 5 splits on HA and running really well.

One issue that’s come up is the minimum selectable temperature on Faikin is +16°C, whereas my remotes for each split will go to +12°C in heat mode.

Any suggestions, either on the Faikin or HA side to have the thermostat card be able to control this?

Thanks!

**Comment by revk** [comment, marked as answer]
Did changing the min temp in the settings not help?

**Reply by Rev-777** [reply]
Well don’t I feel silly.

Apologies and thank you again for the project.
